### PR TITLE
TimeoutStopSecs should be TimeoutStopSec

### PR DESCRIPTION
**Pull request by gnat**
Typo. See: https://www.freedesktop.org/software/systemd/man/systemd.service.html#TimeoutStopSec=

### CODE REVIEW

**Comment by jseldess** [comment]
Sorry for the delay, @gnat! And thanks for the fix.

@taroface, PTAL. If you approve the change, after merging this, please follow-up with a PR porting them to 21.2 docs as well.

**Review by taroface — APPROVED** [review]
LGTM. Thank you for the fix!

**Pull request merged**
